cupyx/scipy/special/_exp1.py

```diff
@@ -0,0 +1,65 @@
+# This source code contains SciPy's code.
+# https://github.com/scipy/scipy/blob/main/scipy/special/specfun/specfun.f
+
+from cupy import _core  # NOQA
+
+
+math_constants_and_eul = """
+#include <cupy/math_constants.h>
+__constant__ double EUL = 0.5772156649015328;
+"""
+
+exp1_definition = """
+template <typename T>
+static __device__ T exp1(T x)
+{
+    if (x == 0) {
+        return CUDART_INF;
+    } else if (x <= 1) {
+        T e1 = 1.0;
+        T R = 1.0;
+
+        for(int k=1; k<=25; k++)
+        {
+            int den = (k + 1) * (k + 1);
+            R = -R*k*x/den;
```

Comment:
```suggestion
            R = -R * k * x / den;
```

Please keep the format consistent in all the lines & files

Reply:
Oh, sorry, my bad, didnt think about it. Fixed!
Is there any guide or maybe formatter, that you would recommend for c++ code?
I used [this](https://codebeautify.org/cpp-formatter-beautifier) formatter